backend/routes/user/user_db.py

```
@@ -200,28 +207,23 @@ def process_simulation_results(sim, league_name: str, active: bool = None) -> di
     return result_data
 
 
-# routes/user/user_db.py
-
+def get_all_leagues(session: Session):
+    """Get all leagues"""
+    leagues = session.exec(select(League).where(League.deleted_date == None)).all()
```

Ruff (0.8.2)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)